lib/setup-lib.bash

```diff
@@ -116,7 +116,7 @@ function installed_direnv() {
       ;;
     latest | LATEST)
       run_cmd asdf install direnv latest
-      version="$(asdf list direnv | tail -n 1 | sed -e 's/ //g')" # since `ASDF_DIRENV_VERSION=latest asdf which direnv` does not work
+      version="$(asdf list direnv | tail -n 1 | sed -e 's/ \|\*//g')" # since `ASDF_DIRENV_VERSION=latest asdf which direnv` does not work
```

Comment:
Would a character class like `'s/[ *]//g'` work here instead?

Reply:
`'s/[ *]//g'` looks better and works fine!  
I changed the code.